spec.html

```diff
@@ -46788,8 +46794,11 @@ <h1>Iterator.prototype.every ( _predicate_ )</h1>
         <emu-alg>
           1. Let _O_ be the *this* value.
           1. If _O_ is not an Object, throw a *TypeError* exception.
-          1. If IsCallable(_predicate_) is *false*, throw a *TypeError* exception.
-          1. Let _iterated_ be ? GetIteratorDirect(_O_).
+          1. Let _iterated_ be the Iterator Record { [[Iterator]]: _O_, [[NextMethod]]: *undefined*, [[Done]]: *false* }.
```

Comment:
this line is only needed inside the if branch below; should it be moved down? (same throughout)

Reply:
Generally speaking we don't declare a variable inside a condition and then also re-declare it outside of the condition.

Could pick a different name, but that seems worse than this.